README.md

```diff
@@ -64,3 +64,35 @@ and [knowledge base][12]. You can also visit our
 [18]: https://codecov.io/github/DataDog/integrations-core?branch=master
 [19]: https://readthedocs.org/projects/datadog-checks-base/badge/?version=latest
 [20]: https://datadog-checks-base.readthedocs.io/en/latest/?badge=latest
+
+## GPG public keys
```

Comment:
One nit: can we move this section above the list of markdown link reference, right after the "Reporting issue" paragraph?

Reply:
done!